src/inputs/Table/Table.js

```diff
@@ -190,31 +252,78 @@ export const Table = (props) => {
         isLoading={isLoading}
         onImport={onImport}
         onExport={onExport}
+        onAddRow={onAddRow ? addRows : null}
+        onDeleteRow={onDeleteRow ? deleteRows : null}
```

Comment:
Unless we mean something specific with `null` in this component, we should probably default to `undefined` instead

Reply:
DONE